svelte/components/Nft/Nft.svelte

```diff
@@ -246,7 +233,7 @@
 
         <ul class="steps">
           <li>
-            <div class="flex"><span class="label">Sell direclty on Opensea</span></div>
+            <div class="flex"><span class="label">Create a link to Opensea</span></div>
```

Comment:
View  ?

Reply:
i was thinking that when we click on the button, it just copy the shortcode that permit to create a link to OpenSea but it's not a link . whatever